test/livebook/live_markdown/markdown_helpers_test.exs

```diff
@@ -24,16 +34,56 @@ defmodule Livebook.LiveMarkdown.MarkdownHelpersTest do
       assert markdown == reformat(markdown)
     end
 
+    test "inline code with a line break" do
+      markdown = "The Game, `Mrs\nHudson`, is on!"
+      assert markdown == reformat(markdown)
+    end
+
+    test "inline code with extra spaces" do
+      markdown = "The Game, `Mrs Huds  on`, is on!"
+      assert markdown == reformat(markdown)
+    end
+
     test "combined" do
       markdown = "The Game, ~~***`Mrs Hudson`***~~, is on!"
       assert markdown == reformat(markdown)
     end
 
+    test "inline math" do
+      markdown = "The Game, $Mrs_{ij} Hudson_{ij}$, is on!"
+
+      assert markdown == reformat(markdown)
+    end
+
+    test "alternate brackets for inline math" do
+      markdown = "The Game, \\(Mrs_{ij} Hudson_{ij}\\), is on!"
+
+      assert markdown == reformat(markdown)
```

Comment:
Fails because the `\` disappear on formatting, changing the meaning

Reply:
Note that this style of math-formatting is not supported by default in Livebook, but _is_ the default for tools like KaTeX's [auto-renderer](https://katex.org/docs/autorender.html).

MathJax's documentation [explains](http://docs.mathjax.org/en/latest/basic/mathematics.html#tex-and-latex-input) that the `$...$` syntax isn't enabled by default to avoid conflicts with text that contains lots of dollar-denominated numbers (avoiding "accidental math" in a sentence like `I paid Bob $200 on Tuesday and $300 on Friday").

Would converting `\(` etc on import to `$` make sense?

Reply:
Since only dollars are supported while rendering, we probably don't have to be concerned with importing `\(`.